realsense2_camera/config/d455_skynode.yaml

```diff
@@ -31,7 +31,7 @@ camera1:
         pointcloud_texture_stream: RS2_STREAM_COLOR
 
         stereo_module:
-          emitter_enabled: true
+          emitter_enabled: 0
```

Comment:
not `false` ?

Reply:
It was complaining about type `bool`. Internally it apparently expects an `int`. 
And for outdoor setups it makes sense to disable it.